### ISSUE TITLE
Unable to remove Torch Effect when Torch is consumed upon use

### ISSUE THREAD

**Issue by jocolamarco** [DFreds/dfreds-convenient-effects#54]
**Describe the bug**
In order to properly address the consumption of Torches, the setting "Destroy on Empty" is set-up in the Item Details information. However, after activating the Torch, if I want to remove the effect before 1 hours has passed, this is not possible and an error message is shown

**To Reproduce**
1. Go to Torch Item in Inventory
2. Click on "Edit Item"
3. Navigate to tab "Details"
4. Scroll-down to Consumable Usage
5. Enable "Destroy on Empty"
6. Close Item window
7. Activate Torch on user
8. DFred's Convenient Effect applied and Lighting effect triggered
9. In the left bar, click the trash icon for "Remove All Convenient Effects" with token selected
10. Verify Error Message is shown and Effect is not removed.

**Expected behavior**
Expect the option "Remove All Convenient Effects"  to remove effects applied on the token

**Screenshots**
![imagem](https://user-images.githubusercontent.com/45291719/131317624-76d6f50d-90c8-4885-9855-a572035995a6.png)

![imagem](https://user-images.githubusercontent.com/45291719/131317324-0da397d1-373a-43b6-a916-276954ea970b.png)

![imagem](https://user-images.githubusercontent.com/45291719/131318067-3b43a38d-71d8-49bb-aee0-ea1dce07c7e5.png)

**Additional context**
**FVTT** 0.8.8
**DND5e** 1.4.2
**DFred's Convenient Effects** 1.9.2
**ATL** 0.2.17

**Comment by DFreds** [comment]
Thanks for bringing this up. I'll try to reproduce and fix it

**Comment by DFreds** [comment]
I'm not able to reproduce this issue. Are you using midi-qol to automatically apply the torch effect when the item is used via the "Apply Convenient Effects" setting in midi? Because if so, it's possible midi fixed something between your bug report and now. Let me know

**Comment by jocolamarco** [comment]
Yes, I'm using that method. Let me double check the versions and update if needed.

current versions:
midiqol = 0.8.47
dae = 0.8.46

**Comment by DFreds** [comment]
Closing this for now since I wasn't able to reproduce the issue.
